### ISSUE TITLE
Ability to run filters by query

### ISSUE THREAD

**Issue by samtux** [t-rex-tileserver/t-rex#111]
Is it possible to perform filters or queries from the server side? I found this example, where the tiles are shown according to the query parameters:

[https://azavea.github.io/tilegarden/demo/client-filter.html](https://azavea.github.io/tilegarden/demo/client-filter.html)

**Comment by pka** [comment]
No, this not possible with query parameters. When using MB GL JSON styling, declarative layer filters can be included in the JSON style. But you would need some code for user defined query parameters.

**Comment by samtux** [comment]
Thank you @pka, it would be an interesting feature of this excellent tool.
